index.html

```diff
@@ -92,6 +92,7 @@ <h2 class="d2l-heading-3">Components</h2>
 		<li><a href="components/switch/demo/switch.html">d2l-switch</a></li>
 		<li><a href="components/tabs/demo/tabs.html">d2l-tabs</a></li>
 		<li><a href="components/tooltip/demo/tooltip.html">d2l-tooltip</a></li>
+		<li><a href="components/breadcrumbs/demo/breadcrumbs.html">d2l-breadcrumbs</a></li>
```

Comment:
Can you put this higher so that it's alphabetical?

Reply:
Done ✔️